BookingsApi/BookingsApi/Controllers/V1/HearingsController.cs

```diff
@@ -467,7 +467,7 @@ private IActionResult ModelStateErrorLogger(string key, string exception, string
             foreach (var command in commands)
             {
                 // dbcontext is not thread safe. loop one at a time
-                var hearing = await _bookingService.SaveNewHearingAndPublish(command, true);
+                var hearing = await _bookingService.SaveNewHearing(command, true);
```

Comment:
can you explain what this change is for? The hearing needs to be published for it to be created in video api

Reply:
I have refactored it

Reply:
I don't think this is the right solution. By changing this to use the CreateConferenceProcess, it makes the FirstdayOfMultidayHearingAsynchronousProcess redundant, and means that notifications will no longer be sent out as per the logic in that process. This will impact the scenario when the new notify templates are switched off, too.

I think it's worth looking at an alternative change that only impacts the new templates for LIPs